code/game/gamemodes/bloodbrother/brother.dm

```diff
@@ -0,0 +1,174 @@
+/datum/antagonist/brother
```

Comment:
[Documentation](https://github.com/ParadiseSS13/Paradise/blob/master/.github/AUTODOC_GUIDE.md) for this file would be appreciated.